### PR TITLE
Fix rendering when swiping workspaces

### PR DESCRIPTION
**Pull request by thejch**
#### Describe your PR, what does it fix/add?
Fixes https://github.com/hyprwm/Hyprland/issues/5361

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)
No

#### Is it ready for merging, or does it need work?
Hopefully ready

### CODE REVIEW

File: src/render/Renderer.cpp [hunk 1]
```diff
@@ -225,13 +225,15 @@ bool CHyprRenderer::shouldRenderWindow(CWindow* pWindow, CMonitor* pMonitor) {
...
-        if (PWINDOWWORKSPACE->m_vRenderOffset.isBeingAnimated() || PWINDOWWORKSPACE->m_fAlpha.isBeingAnimated() || PWINDOWWORKSPACE->m_bForceRendering) {
+        if (PWINDOWWORKSPACE->m_vRenderOffset.isBeingAnimated() || PWINDOWWORKSPACE->m_fAlpha.isBeingAnimated() || PWINDOWWORKSPACE->m_bForceRendering)
             return true;
-        } else {
-            if (PWINDOWWORKSPACE->m_bHasFullscreenWindow && !pWindow->m_bIsFullscreen && !pWindow->m_bIsFloating && !pWindow->m_bCreatedOverFullscreen &&
-                pWindow->m_fAlpha.value() == 0)
-                return false;
-        }
+
+        if (PWINDOWWORKSPACE->m_bHasFullscreenWindow && !pWindow->m_bIsFullscreen && !pWindow->m_bIsFloating && !pWindow->m_bCreatedOverFullscreen &&
```
Comment (vaxerski) [inline src/render/Renderer.cpp:231]: > `!pWindow->m_bIsFloating && !pWindow->m_bCreatedOverFullscreen`

vvv
`(!pWindow->m_bIsFloating || !pWindow->m_bCreatedOverFullscreen)`

Comment (thejch) [reply src/render/Renderer.cpp:231]: why not just `!pWindow->m_bCreatedOverFullscreen`
can tiled windows even be `m_bCreatedOverFullscreen`?

Comment (vaxerski) [reply src/render/Renderer.cpp:231]: idk better safe than sorry

Comment (thejch) [reply src/render/Renderer.cpp]: fixed

**Review by vaxerski — APPROVED** [review]

**Pull request merged**